openedx/core/djangoapps/user_api/urls.py

```diff
@@ -12,17 +12,22 @@
 urlpatterns = patterns(
     '',
     url(
-        r'^v1/accounts/' + USERNAME_PATTERN + '$',
+        r'^v1/accounts/%s$' % (USERNAME_PATTERN,),
```

Comment:
Preference for `r'^v1/accounts/{}$'.format(USERNAME_PATTERN)` vs. `r'^v1/accounts/{username}$'.format(username=USERNAME_PATTERN)` ?